tools/file_packager.py

```diff
@@ -291,6 +291,14 @@ def main():
   }
   Module.expectedDataFileDownloads++;
   (function() {
+   // When running as a pthread, FS operations are proxied to the main thread, so we don't need to
+   // fetch the .data bundle on the worker
+   if (Module['ENVIRONMENT_IS_PTHREAD']) {
```

Comment:
Are we sure that `ENVIRONMENT_IS_PTHREAD` is always exported on the Module object in pthreads mode?

Reply:
It should be guaranteed, since we rely on it further down to set `ENVIRONMENT_IS_PTHREAD` 
https://github.com/emscripten-core/emscripten/blob/0ca340670908acb5ac917fa123ce24c6443bca56/src/shell.js#L125